demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -350,9 +359,23 @@ export function ProductsGrid() {
         },
     ];
 
+    const convertCustomFilters: ConvertCustomFilterCallback = (filterItem: GridFilterItem, columns: GridColDef[], filterModel?: GridFilterModel) => {
+        if (filterItem.field === "titleSlugOrDescription") {
+            return {
+                or: [
+                    { title: { contains: filterItem.value } },
+                    { slug: { contains: filterItem.value } },
+                    { description: { contains: filterItem.value } },
+                ],
+            } as GqlFilter;
```

Comment:
filterITem.operator is not respected.

I know this is just an example, still I'd like to see how this would look like in real world. (besides, demo should be fully implemented with no bugs (in theory, i know it has many bugs) - because it might be used as source for best pratice)